docs/preview/connected-registry/quickstart-deploy-connected-registry-nested-iot-edge-cli.md

```diff
@@ -0,0 +1,273 @@
+  "ACR_REGISTRY_LOGIN_SERVER": "<Optional: connected registry login server. More info at https://aka.ms/acr/connected-registry>"
+}
+```
+
+The JSON above lists the environment variables that need to be passed to the connected registry container at run time. The following environment variables are optional:
+
+- `ACR_REGISTRY_LOGIN_SERVER` - this is the hostname or FQDN of the IoT Edge device that hosts the connected registry.
+
+You will need the information for the IoT Edge manifest below.
+
+  > [!IMPORTANT]
+  > Make sure that you save the generated connection string. The connection string contains one-time password that cannot be retrieved. If you issue the command again, new passwords will be generated. You can generate new passwords using the [az acr token credential generate][az-acr-token-credential-generate] command.
+
+## Configure a deployment manifest for the nested IoT Edge
+
+A deployment manifest is a JSON document that describes which modules to deploy to the IoT Edge device. For more information about how deployment manifests work and how to create them, see [Understand how IoT Edge modules can be used, configured, and reused](../iot-edge/module-composition.md).
+
+To deploy the connected registry module using the Azure CLI, save the following deployment manifest locally as a `.json` file. 
+
+[!IMPORTANT] In the bellow deployment manifest, the IP address `10.16.7.4` is the IP address of the device hosting parent connected registry. Make sure you replace this IP address with the one your parent device uses.
+
+```json
+{
+    "modulesContent": {
+        "$edgeAgent": {
+            "properties.desired": {
+                "modules": {
+                    "connected-registry": {
+                        "settings": {
+                            "image": "10.16.7.4/acr/connected-registry:0.2.0",
+                            "createOptions": "{\"HostConfig\":{\"Binds\":[\"/home/azureuser/connected-registry:/var/acr/data\",,\"/usr/local/share/ca-certificates:/usr/local/share/ca-certificates\",\"/etc/ssl/certs:/etc/ssl/certs\"]}}"
+                        },
+                        "type": "docker",
+                        "env": {
+                            "ACR_REGISTRY_CONNECTION_STRING": {
+                                "value": "ConnectedRegistryName=myconnectedmirror;SyncTokenName=myconnectedmirror-sync-token;SyncTokenPassword=s0meCoMPL3xP4$$W0rd001!@#;ParentGatewayEndpoint=10.16.7.4;ParentEndpointProtocol=https"
+                            }
+                        },
+                        "status": "running",
+                        "restartPolicy": "always",
+                        "version": "1.0"
+                    },
+                    "IoTEdgeApiProxy": {
+                        "settings": {
+                            "image": "10.16.7.4/azureiotedge-api-proxy:latest",
```

Comment:
`azureiotedge-api-proxy` needs to included in the repo list when creating connected registry.